pkg/solana/chainwriter/chain_writer.go

```diff
@@ -495,6 +562,31 @@ func getLookupTableAddresses(ctx context.Context, client client.MultiClient, tab
 	return alt.Addresses, nil
 }
 
+func CalculateTxSize(tx *solana.Transaction) (int, error) {
+	if tx == nil {
+		return 0, errors.New("tx is nulll")
+	}
+	copyTx := utils.DeepCopyTx(*tx)
+
+	// Set instructions and fields that are added further downstream with arbitrary values to get an accurate tx size
```

Comment:
What happens if you don't set these? It's unclear to me what ComputeUnit price and limit have to do with tx size